modules/fedimint-ln-client/src/lib.rs

```diff
@@ -1562,3 +1626,17 @@ async fn set_payment_result(
             .await;
     }
 }
+
+fn tweak_user_key<Ctx: Verification + Signing>(
```

Comment:
Let's add a docstring here too. Would probably be helpful to explain why we need the key tweak